gallerypage.py

```
@@ -1144,36 +1046,73 @@ def _write_jekyll(self, filepath, remote_keys, replacements,
+                        'value': value[0].encode('ascii', 'xmlcharrefreplace').decode('ascii'),
+                        'additional_values': value[1].encode('ascii', 'xmlcharrefreplace').decode('ascii')
+                    }
+                    
+                yaml_data['catalog_table'].append(row)
+                
+            yaml_output = yaml.dump(yaml_data, Dumper=FixIndent, default_flow_style=False, sort_keys=False)
+            yaml_output = f"---\n{yaml_output}---\n"
+            f.write(yaml_output)
+            # Include small image with a link to a larger one
+            f.write('# {{ page.title }}\n\n')
+            f.write('[ ![{{ page.title }}]({{ site.baseurl}}%s){:target="_blank"}]({{ site.baseurl}}%s){:target="_blank"}{: .press_release}\n\n' % (self.local_small_url, self.local_medium_url))
```

Ruff (0.14.14)

Use format specifiers instead of percent format

Replace with format specifiers

(UP031)